.gitignore

```diff
@@ -2,6 +2,8 @@
 *.pyc
 *.pyo
 **/__pycache__/
+**/.project
```

Comment:
Personnaly I ignore eclipse files globally on my workstation using https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
+1 to don't have IDE stuff in ignore in the project